subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -50,6 +50,21 @@ Gradle 8.4 now configures XML parsers with security features enabled.
 If your build logic has dependencies on old XML parsers that don't support secure parsing, your build may now fail.
 If you encounter a failure, check and update or remove any dependency on legacy XML parsers.
 
+If you are unable to upgrade, you can disable resolving XML external entities by setting JAXP system properties.
+This can be made by adding the following lines in a `gradle.properties` file:
+
+```
+systemProps.javax.xml.accessExternalDTD=false
+systemProps.javax.xml.accessExternalSchema=false
+systemProps.javax.xml.accessExternalStylesheet=false
+```
+
+Another possible workaround is to instruct the JVM to use the native provider. For example, in OpenJDK this can be done by adding the following to `gradle.properties`:
```

Comment:
```suggestion
Another possible workaround is to instruct the JVM to use the native provider. 
For example, in OpenJDK this can be done by adding the following to `gradle.properties`:
```